content/2022-06-03-brigade-2.6.0-and-friends.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Brigade 2.6.0 and Friends"
+slug: "2022-06-03-brigade-2.6.0-and-friends"
+description: "Brigade 2.6.0 has been released"
+date: "2022-06-03"
+author: "Kent Rancourt"
+summary: "Brigade 2.6.0 and several new peripherals have been released!"
+---
+
+Hello, Brigadiers! We hope everyone's had a great week! It's certainly been a
+great one for us because we're closing it out with the release of Brigade v2.6.0
+and a few new peripherals!
+
+Brigade v2.6.0 continues the trend of small, frequent releases that are more
+evolutionary than revolutionary. Apart from the bug fixes, doc improvements, and
+other maintenance that go into every release, there's just one noteworthy change
```

Comment:
```suggestion
other maintenance that goes into every release, there's one noteworthy change
```

Reply:
On board with striking the "just," but wasn't "go" grammatically correct in this sentence?